### PR TITLE
Acknowledge community contributions in Changelog

### PR DESCRIPTION
**Pull request by antkmsft**

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,14 @@
 
 ### Other Changes
 
+- [[#6014]], [[#6016]] Fixes for Doxygen generation. (A community contribution, courtesy of _[chewi](https://github.com/chewi)_)
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:15]: Add the PR links.

```suggestion
- [[#6014]](https://github.com/Azure/azure-sdk-for-cpp/pull/6014), [[#6016]](https://github.com/Azure/azure-sdk-for-cpp/pull/6016) Fixes for Doxygen generation. (A community contribution, courtesy of _[chewi](https://github.com/chewi)_) 
```

**Review by ahsonkhan — APPROVED** [review]
LGTM

**Review by LarryOsterman — APPROVED** [review]

**Comment by antkmsft** [comment]
Thank you, @chewi!

**Comment by chewi** [comment]
A contribution done by an MS employee in work time that (indirectly) benefits Flatcar doesn't quite feel like a community contribution to me, but you still handled it like one, and I believe that is the most important thing, so _thank you_! :slightly_smiling_face:

**Pull request merged**